src/commands/sahko.rs

```diff
@@ -0,0 +1,10 @@
+use teloxide::types::InputFile;
+
+use crate::services::porssisahko::get_price_chart;
+
+pub async fn get_sahko_inputfile() -> InputFile {
```

Comment:
I suppose this should be named more according to command, instead of what it does?